go.mod

```diff
@@ -0,0 +1,11 @@
+// change the module path to your directory
```

Comment:
No need for this comment, as ideally all repository cloned to personal laptops follows the format github.com/ndv6/tsaving